internal/pkg/manifest/validate.go

```diff
@@ -328,9 +346,25 @@ func (t *TaskConfig) Validate() error {
 
 // Validate returns nil if PlatformArgsOrString is configured correctly.
 func (p *PlatformArgsOrString) Validate() error {
+	if err := p.PlatformString.Validate(); err != nil {
+		return err
+	}
 	return p.PlatformArgs.Validate()
 }
 
+// Validate returns nil if PlatformString is configured correctly.
+func (p *PlatformString) Validate() error {
+	if p == nil {
+		return nil
+	}
+	val := string(*p)
+	reg := regexp.MustCompile(`^.+\/.+$`)
+	if !reg.MatchString(val) {
```

Comment:
We should avoid using regular expressions so frequently (see this article by Rob Pike: https://commandcenter.blogspot.com/2011/08/regular-expressions-in-lexing-and.html) 

In this situation our expected string is fairly straightforward and we can get a better performance boost and less error prone code like this:
```go
for _, validPlatform := range validPlatforms {
  if val == validPlatform {
     return nil
  }
}
return fmt.Errorf(`platform %s must be one of: %s`, val, strings.Join(validPatforms, ", "))
```

I like his rule of thumb around regex:
> Another way to look at it is that lexers and parsing are matching statically-defined patterns, but regular expressions' strength is that they provide a way to express patterns dynamically.